shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -67,15 +67,12 @@
   private class JobAttempt {
     private final LoadConfig mJobConfig;
     private final RetryPolicy mRetryPolicy;
-    private final JobMasterClient mClient;
 
     private Long mJobId;
 
-    private JobAttempt(LoadConfig jobConfig, RetryPolicy retryPolicy, ClientContext clientContext) {
+    private JobAttempt(LoadConfig jobConfig, RetryPolicy retryPolicy) {
       mJobConfig = jobConfig;
       mRetryPolicy = retryPolicy;
-      mClient = JobMasterClient.Factory.create(
-          JobMasterClientContext.newBuilder(clientContext).build());
```

Comment:
Why can't we just use this one always?

Reply:
What does that mean?

Reply:
nevermind, I was thinking this was being done above job-attempt.